doc/man3/OSSL_PARAM_int.pod

```diff
@@ -51,8 +103,31 @@ OSSL_PARAM_set_octet_ptr
 
  OSSL_PARAM *OSSL_PARAM_locate(OSSL_PARAM *array, const char *key);
 
- int OSSL_PARAM_get_TYPE(const OSSL_PARAM *p, const char *key, TYPE *val);
- int OSSL_PARAM_set_TYPE(const OSSL_PARAM *p, const char *key, TYPE val);
```

Comment:
Because this is removed, the beginning of the DESCRIPTION becomes less comprehensible:

```
=head1 DESCRIPTION

A collection of utility functions that simplify and add type safety to the
OSSL_PARAM arrays. The following B<TYPE> names are supported:
```

This is *also* a good reason to keep the TYPE variants in the SYNOPSIS section even if you go for specific function names in NAME section (which, as has already been pointed out, is exactly the correct move).